src/BuildScriptGenerator/Php/PhpBashBuildSnippet.sh.tpl

```diff
@@ -0,0 +1,13 @@
+﻿echo "PHP executable: $php"
+
+cd "$SOURCE_DIR"
+
+{{ if ComposerFileExists }}
```

Comment:
if not use bash, `if [-f ]`?

Reply:
a style preference - why run extra commands in the build script if we know for certain they won't be needed?